tdiff_path/dataset_operations/dataset_operations.py

```diff
@@ -318,3 +320,16 @@ def reflection_to_transmission_phase(incoming_data):
     if 'phase_rad' in data.dtype.names:
         new_data['phase_rad'] = np.true_divide(new_data['phase_rad'], 2.0)
     return new_data
+
+
+def get_min_dataset_length(data_dict):
```

Comment:
Funny I thought there would be a function already made to do this. Do you maybe want to elaborate on this function since it seems you are only looking at the antenna dataset? Maybe why do you want the minimum if it is used for a specific case?

Reply:
updated docstring and renamed antenna variable as it may not necessarily be an antenna. I'm not aware of a function that does this already